packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -69,6 +69,27 @@ class DriveCommand extends RunCommandBase {
       ..addFlag('build',
         defaultsTo: true,
         help: 'Build the app before running.',
+      )
+      ..addOption('driver-port',
+        defaultsTo: '4444',
+        help: 'The port where Webdriver server is launched at. Defaults to 4444.',
+        valueHelp: 'n'
+      )
+      ..addFlag('headless',
+        defaultsTo: true,
+        help: 'Whether the driver browser is going to be launched in headless mode. Defaults to True.',
+      )
+      ..addOption('browser-name',
```

Comment:
There is an allowed values option field that will let you restrict this setting

Reply:
Updated.